docs/actions/assert.md

```diff
@@ -0,0 +1,235 @@
+# Assertions
+
+## Assert
+assert that all evaluated inputs are true.
+=== "Inputs"
+
+    | Required | Name            | Type        |     Default     |
+    |:--------:|:----------------|:------------|:---------------:|
+    |    *     | `asserts`       | List of Map |                 |
+
+=== "Outputs"
+    No outputs
```

Comment:
We should decide, do we put this in the output tab or outside the tab ?
ex. https://www.chutney-testing.com/actions/jms/#jms-sender & https://www.chutney-testing.com/actions/ssh/#download

Reply:
I prefer to represent outputs like inputs using tabs